.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Related Issue/Addition to code
+- Issue goes here
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Additional Info
```

Comment:
Please add a newline to separate headers, and use the correct heading level (only using one H1 per document).

```suggestion
## Additional Info

```